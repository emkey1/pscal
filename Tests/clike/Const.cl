const int N = 3 + 2;

int main() {
    int arr[N];
    printf("%d %d\n", N, sizeof(arr));
    switch (N) {
        case 5: printf("C\n"); break;
        default: printf("X\n");
    }
    return 0;
}

