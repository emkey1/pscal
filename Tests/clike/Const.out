5 20
C
